### PR TITLE
1900 dependencies dependencies to rust sections

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes: #1900 
Creates a new section `<dependencies>` with `dependency`s inside. The result XML looking like that:
```
...
  <rusts><!--"Rust inserts"-->
      <rust code="75 73 65 20 72 65 6F 3A 3.."
            loc="Φ.org.eolang.creates-object.r">
         <dependencies>
            <dependency name="72 65 6F 3A 20 30 2E 30 2E 32"/>
            <dependency name="72 61 6E 64 3A 20 30 2E 35 2E 35"/>
         </dependencies>
      </rust>
   </rusts>
</program>
```

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please review. There are build failures because of math in eo-runtime

**Comment by levBagryansky** [comment]
@Graur please take a look.

File: eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl [hunk 1]
```diff
@@ -37,6 +37,17 @@ SOFTWARE.
               <xsl:attribute name="loc">
                 <xsl:value-of select="../attribute(loc)"/>
               </xsl:attribute>
+              <dependencies>
+                <xsl:for-each select="../o/o">
```
Comment (Graur) [inline eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl:41]: @levBagryansky I believe searching the parent node attribute has separate value. Something like this:
```
<xsl:for-each select="ancestor::o[1]/@o">
```

File: eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl [hunk 2]
```diff
@@ -37,6 +37,17 @@ SOFTWARE.
...
+                  <xsl:if test="../attribute(base) = 'org.eolang.tuple' and attribute(base) = 'org.eolang.string'">
```
Comment (Graur) [inline eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl:42]: @levBagryansky Same here. I believe we can replace `..` by `parent::@base`. Or something similar to this approach

**Review by Graur — CHANGES_REQUESTED** [review]

**Review by Graur — APPROVED** [review]
@levBagryansky Looks good to me, thanks!
@yegor256 Please, take a look

**Comment by levBagryansky** [comment]
@yegor256 please take a look

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**